kijiji-query.js

```diff
@@ -42,9 +43,8 @@ var parseXML = function(xml) {
         var ad = {};
         $(item).children().each(function(i, child) {
             ad[child.name] = $(child).text();
-        });    
-    
-        ad.innerAd = {};
```

Comment:
I waffled for a while on whether leaving the empty object was convenient or merely deceptive. I'll put it back.